requester/requester.go

```diff
@@ -29,9 +29,39 @@ import (
 	"sync"
 	"time"
 
+	"github.com/vkuznet/x509proxy"
 	"golang.org/x/net/http2"
 )
 
+// Return client X509 certificates
+func Certs() (tls_certs []tls.Certificate) {
+	uproxy := os.Getenv("X509_USER_PROXY")
+	uckey := os.Getenv("X509_USER_KEY")
+	ucert := os.Getenv("X509_USER_CERT")
+	if len(uproxy) > 0 {
+		// use local implementation of LoadX409KeyPair instead of tls one
+		x509cert, err := x509proxy.LoadX509Proxy(uproxy)
+		if err != nil {
+			fmt.Println("Fail to parser proxy X509 certificate", err)
+			return
+		}
+		tls_certs = []tls.Certificate{x509cert}
+	} else if len(uckey) > 0 {
+		x509cert, err := tls.LoadX509KeyPair(ucert, uckey)
+		if err != nil {
+			fmt.Println("Fail to parser user X509 certificate", err)
```

Comment:
return fmt.Errorf("failed to load X509 key pair set by X509_USER_KEY: %v, err), nil